src/o365/graph/commands/teams/teams-channel-message-list.ts

```diff
@@ -0,0 +1,108 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  channelId: string;
+}
+
+class GraphTeamsChannelMessageListCommand extends GraphItemsListCommand<any> {
```

Comment:
We should use `Message` type instead of `any`

Reply:
Message type representing the full message json response ?

Reply:
Not necessarily the full type but at least the properties that we use in code to ensure that we won't run into things like typos.